.github/workflows/build+test+deploy.yml

```diff
@@ -1,123 +1,71 @@
-name: CI
+name: Build, Test and SelfCheck
 
 on:
-  pull_request:
-  workflow_dispatch:
-
-  # to execute once a day (more info see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule )
-  schedule:
-    - cron: '0 0 * * *'
-
   push:
-    tags:
-      - 'v*'
     branches:
-      - '**'
-
-env:
-  DOTNET_VERSION: 9.0.201
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
@xperiandri doing the above will make PRs not have CI feedback; please revert all these changes in the workflow above